ci/scripts/gen-integration-test-yaml.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+
+import subprocess
+
+CASES_MAP = {
+    'ad-click': ['json'],
+    'ad-ctr': ['json'],
+    'cdn-metrics': ['json'],
+    'clickstream': ['json'],
+    'livestream': ['json', 'protobuf'],
+    'prometheus': ['json'],
+    'schema-registry': ['json'],
+    'mysql-cdc': ['json'],
+    'postgres-cdc': ['json'],
+    'mysql-sink': ['json'],
+    'postgres-sink': ['json'],
+    'iceberg-cdc': ['json'],
+    'twitter': ['json', 'protobuf'],
+    'twitter-pulsar': ['json'],
+    'debezium-mysql': ['json'],
+    'debezium-postgres': ['json'],
+    'debezium-sqlserver': ['json'],
+    'tidb-cdc-sink': ['json'],
+    'citus-cdc': ['json'],
+    'kinesis-s3-source': ['json'],
+    'clickhouse-sink': ['json'],
+    'cockroach-sink': ['json'],
+    'kafka-cdc-sink': ['json'],
+    'cassandra-and-scylladb-sink': ['json'],
+    'elasticsearch-sink': ['json'],
+    'redis-sink': ['json'],
+}
+
+def gen_pipeline_steps():
+    pipeline_steps = ""
+    for key, values in CASES_MAP.items():
+        for value in values:
+            pipeline_steps += f"""
+ - label: Run Demos {key} {value}
+   key: {key}-{value}
```

Comment:
Can we rename `key` to `test_type`, and `values` to `test_format`? Or something more readable